src/components/TopBar/TopBar.jsx

```diff
@@ -1,64 +1,136 @@
zgyBjj/OTVlwSgoYjOBgAAeGsox0v/U04vAUzl/0M6d4gCAAAXOF/6n/KiAjDFUgAAwGFelP6nfEsAGmIpAADgnqE8Kf1PebEEMJX/D+tNdgUAiIY3pf8prxIASTLGvJL0ynIYAABMOXfT3yK8WgKYyg8I+k3cFQAAsGsoz0r/U95VACQOCAIAOONLHwd/ydMEQOKuAACAdXv5WOQlL5cAZqVp+pukDdtxAACi4uw1v4vytgIw43NJY9tBAACiMZZ013YQ6/I+AeCUQABAxfZ8Xfef5f0SwFSapj9I+tp2HACAoHl12t9VQkoA6mJrIACgPENJn+Y70bzn/RLAVP5/yF3RDwAAKN5Y2X7/seU4ChNMAiDRDwAAKE0Q6/6zgkoAJMkY84M4HwAAUJwf87ElKMH0AMzK+wHeiPMBAADr8X6//2WCqwBI5/0AnA8AAFjHUAHs979MkAmAdN4P8LntOAAA3vL2nP9FBJsASOf3BXxjOw4AgHe8Pud/EUH2AFyUpumRpC9sxwEA8MKPxpgntoMoWywJQF00BQIArhds099FQS8BTM00BQ7tRgIAcNhQATf9XRRFBWAqTdMNZZWAut1IAACOGSs75ndoOY7KRFEBmDLGnIqmQADAh4Lu+J8nqgRAkowxR+K4YADAO3vGmFe2g6haVEsAs7g+GACgSDr+54k2AZCkNE3fSGrbjgMAYMUrY0y0B8bFngDUxfZAAIjRUFnT39hyHNZE1wMwi+2BABCloaS7MQ/+UuQJgHR+Z8BdkQQAQAyGygb/oeU4rIs+AZDeuzhobDcSAECJxpI+Z/DPRN0DcFGapm1lPQEAgPB8mp8HA1EBeE9+89OXtuMAABTuSwb/95EAXJAfFMRpgQAQjm/y93bMIAGYwxjzgzgtEABCsJe/p+MCEoBLGGN2RRIAAD7by9/LMQcJwBVIAgDAWwz+1yABuAZJAAB4h8F/ASQACyAJAABvMPgviARgQSQBAOA8Bv8lkAAsgSQAAJzF4L8kEoAlkQQAgHMY/FdAArACkgAAcAaD/4pIAFZEEgAA1jH4r4HLgNaUpukTSd/bjgMAIvMNJ/ythwSgAGmaPpb0k+04ACASX3K2//pIAAqSpumGsquE63YjAYBgjSV9nt/cijXRA1CQ/JrJu5KGdiMBgCANJd1l8C8OFYCCpWnaUFYJaNiNBACCMVQ2+A8txxEUKgAFy/+BUgkAgGKcisG/FCQAJcj/oX4q6ZXdSADAaz0x+JeGBKAkxpixMeZzST/ajgUAPPSjMeauMWZsO5BQkQCUzBjzRBwYBADL2MvfO1EimgArkqbpA2VnBdTtRgIAzhorO+DnyHIcUSABqBA7BADgUkNle/xPLccRDZYAKsQOAQCY61RZs9+p5TiiQgJQMWPM0BhzWzQHAoAk/Sw6/a0gAbCE5kAA0DfGmMd0+ttBD4BlaZq2lTUHNuxGAgCVGYsz/a0jAXAAzYEAInKqbPAfWo4jeiwBOIC+AACR+FGs9zuDBMAheV/AN8rKYwAQirGy9f4nrPe7gyUAB7EkACAgQzHrdxIVAAfNXCb0s+VQAGAdP0r6lMHfTSQAjsovE3oslgQA+GcsSv7OYwnAAywJAPDIqejy9wIVAA/M7BLg4CAALvvRGEPJ3xNUADzDwUEAHDSU9CUH+/iFCoBn8hfYXdEgCMANr5Q1+vUsx4ElUQHwWJqmjyXtiGoAgOqNlc36X1mOAysiAfBc3iC4K+kLu5EAiEhP2eA/tBwH1sASgOfyBsHHkr5Utg4HAGUZK9vex8E+AaACEBCqAQBK1BOz/qCQAASI3gAABRqLtf4gsQQQIGPMkdgpAGB9P0q6zeAfJioAgUvTdEPSr6IaAGBxQ7GvP3hUAAJnjDnlFEEACxpL2jPG3GbwDx8VgIjQJAjgCj3R5BcVEoAI0SQIYMZQlPujRAIQsTRNd5VVAxp2IwFgwVjZ5T27luOAJSQAkWNZAIjSj5J2jTFj24HAHhIASDpPBH6S1LYbCYAS9ZQ1+fUsxwEHsAsAks6PFL4rjhQGQnQq6W5+hG/PcixwBBUAzEWjIBCEsaQnxhgOBcMHSABwJRIBwEtjZev8P7DOj8uQAOBaeX/AY7FjAHDdWAz8WBAJABZGIgA4aywGfiyJBABLIxEAnDEWAz9WRAKAlZEIANaMxcCPNZEAYG15ItAWzYJA2cZi4EdBSABQKHYNAKUYSvpB0s8M/CgKCQBKkScCX4iTBYF19MTJfSgJCQBKlabphqQn4q4BYBmvlF3U07McBwJGAoBKzFw6dEcsDwDzjMX6PipEAoDKsTwAvKcn6UjSPxn4USUSAFgzUxX4u6S6zViAio0l/SzpFWV+2EICAOvSNK1LeiCqAghfT9n6Pt38sI4EAE6hVwABGitb239ljDm1GwrwDgkAnJWm6QO9qwwAPhkrm+3TyQ9nkQDAeSwRwCM9UeKHJ0gA4JWZY4dJBuCKnhj04SESAHiLZAAW9cSgD8+RACAIM8nA3/P/1u1FgwCNJZ3q3da9sc1ggCKQACBIaZq2lV1VzG4CrGoo6Z/KZvqnDPoIDQkAgpffR3BHWSNh22YscNpY2Sz/lbJT+YYWYwFKRwKA6OTVgQfKkoINm7HAulNJb8UsHxEiAUDU8i2G7fyDhCB8p8oG/J6kHgM+YkYCAMzIE4INvZ8Q1G3Fg7WMlQ34/xYzfOADJADANfIegg1lScFfRZXAVaczH285dhe4GgkAsIK8j+Cvkm7rXVJQtxdRVMbKOvRP9a6kP2R2DyyHBAAoSF4paChLBv468zlWM9b7A/1/lJXxh7YCAkJCAgCULE8M6npXMfhYWXLQEFWDsd6t1f9H0h9ioAcqQQIAWDTTdFhXlhh8pCxRqOcfDfmbJIz1bhY/lPQ/vRvgh6JsD1hFAgB4ID/quH7ho5H/+OP8v42ZX5k+RnN+dp1x/jFrOOdn/7nws+H058zeAff9P7T/5yGeoKUcAAAAAElFTkSuQmCC";
```

Comment:
This is useful in other places. Can you abstract it as a helper method in `helpers.js`

Reply:
yep can do